verification.py

```diff
@@ -724,27 +738,34 @@ def _discoal_str_to_simulation(self, args):
                 theta = float(tokens[i + 1])
             if tokens[i] == "-r":
                 rho = float(tokens[i + 1])
-        mod_list = [None]
+        mod_list = [("hudson")]
         if alpha is not None:
             # sweep model
             mod = msprime.SweepGenicSelection(
-                position=sweep_site,
+                position=np.floor(sweep_site * seq_length),
                 start_frequency=1.0 / (2 * refsize),
                 end_frequency=1.0 - (1.0 / (2 * refsize)),
-                alpha=alpha,
-                dt=1.0 / (40 * refsize),
+                alpha=alpha * 2,  # discoal fitness model is 1, 1+s, 1+2s
+                dt=1e-6,
             )
             mod_list.append((sweep_mod_time, mod))
+            # if an event is defined from discoal line
+            # best thing to do is rescale to Ne=0.25
+            # so that time scale are consistent
+            # see note at msprime/cli.py line 626
+            # and following for alternate solution
+            if sweep_mod_time > 0:
+                refsize = 0.25
         # append final model
-        mod_list.append((None, None))
+        mod_list.append((None, "hudson"))
```

Comment:
Do we need to make this change, or are you doing this to make it more readable?

Reply:
yes trying to make this easier for others to read and evaluate